src/flexmeasures_client/client.py

```diff
@@ -101,20 +101,26 @@ async def request(
         self.start_session()
 
         polling_step = 0  # reset this counter once when starting polling
-        self.reauth_once = True
+        # we allow retrying once if we include authentication headers
+        reauth_once = self.reauth_once if include_auth else False
```

Comment:
Do we want users to be able to turn of the reauthentication? otherwise we can just use `True` instead of self.reauth_once here

Reply:
I can't think of a use case either. I'll remove it from the initialization settings.